src/Backend/Modules/Blog/Layout/Templates/Index.html.twig

```diff
@@ -43,53 +43,31 @@
   {% endif %}
 
   {% if dgRecent %}
-    <div class="row">
-      <div class="col-md-12">
-        <div class="content-block content-block-pb">
-          <div class="content-title">
-            <p>{{ 'lbl.RecentlyEdited'|trans|ucfirst }}</p>
-          </div>
-          <div class="table-responsive">
-            {{ dgRecent|raw }}
-          </div>
-        </div>
+    <div class="content-block content-block-pb">
+      <div class="content-title">
+        <p>{{ 'lbl.RecentlyEdited'|trans|ucfirst }}</p>
       </div>
+      {{ macro.dataGrid(dgRecent, 'msg.NoItems'|trans|format(geturl('add')) }}
```

Comment:
well there isn't a single place in the backend which uses capitalised parameters in the `geturl()` modifier. This line of code in the BackendModel::createURLForAction() will convert everything anyways:

```
$action = mb_strtolower(preg_replace('/([a-z])([A-Z])/', '$1_$2', $action));
```

`add` stays `add`
`Add` becomes `add`
`AddWhatever` becomes `add_whatever`
and so on..